src/gui/test/screen_test_graph.c

```diff
@@ -86,7 +86,7 @@ int screen_test_graph_event(screen_test_term_t *screen, window_t *window, uint8_
     } else if (event == WINDOW_EVENT_CLICK) {
         switch ((int)param) {
         case 1:
-            //screen_open(pscreen_menu_service->id);
+            //screen_open(get_scr_menu_service()->id);
```

Comment:
Why do we have this commented code here?

Reply:
there is more commented code in this file,
not my code to remove